ci-operator/step-registry/ipi/conf/manual-creds/remove-unnecessary-creds/ipi-conf-manual-creds-remove-unnecessary-creds-ref.yaml

```diff
@@ -0,0 +1,17 @@
+ref:
+  as: ipi-conf-manual-creds-remove-unnecessary-creds
+  from_image:
+   namespace: ocp
+   name: "4.12"
+   tag: upi-installer
+  commands: ipi-conf-manual-creds-remove-unnecessary-creds-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
```

Comment:
I think this step need to  set `BASELINE_CAPABILITY_SET` and `ADDITIONAL_ENABLED_CAPABILITY_SET` to receive the values passed from workflow level.

Reply:
Updated.